src/app/monitoring/containers/search-logs/search-logs.component.ts

```diff
@@ -78,8 +78,25 @@ export class SearchLogsComponent implements OnInit {
     this.error = false;
 
     this.fireRequest(request).subscribe((data: SearchResponse) => {
+
       this.hits = data.hits;
       this.hitsSubject.next(this.hits);
+    }, (error) => {
+      // get forwarded elastic search error
+      var ese = error.error.error;
+      if (ese){
+        // this line makes an error hint show up on gui, telling the user to check his search input
+        this.error = true;
+        // detailled error information is not shown to the user, but it could easily be done
+        var errorType = ese.failed_shards[0].reason.caused_by.caused_by.type;  
+        var errorReason = ese.failed_shards[0].reason.reason;
+        var errorReasonDetail = ese.failed_shards[0].reason.caused_by.caused_by.reason;
```

Comment:
Don't use var. Use let and const instead. Also comment this, since there is no benefit and we allocate memory for nothing.